sdk/src/CrashReporter.ts

```diff
@@ -200,6 +208,48 @@ export default class CrashReporter {
       return;
     }
 
+    if (isFatal) {
+      setCurrentTags(getCurrentTags().concat('UnhandledError'))
+    }
+
+    const stack = await this.cleanStackTrace(error);
+
+    const payload = await this.generateCrashReportPayload(error, stack);
+
+    this.managePayload(payload);
+  }
+
+  /**
+   * Processes a manually sent error (using local tags, not global).
+   * @param error - The Error to be processed.
+   * @param params
+   */
+  async processManualCrashReport(error: Error, params: any[]) {
```

Comment:
I don't like how the `params` parameter has a type of "any". This would then allow types which are not acceptable such as, functions or numbers (which may break).